### PR TITLE
[AND-3760] update admob to vungle 6.10.3

### PR DESCRIPTION
**Pull request by jiancchen**
Closed on PR:
https://github.com/Vungle/googleads-mobile-android-mediation/pull/111/files

6.10.3 Changes based off master.

1) Updated to 6.10.3
2) If coppa in admob adapter is set, we will use that value

### CODE REVIEW

File: ThirdPartyAdapters/vungle/vungle/build.gradle [hunk 1]
```diff
@@ -8,17 +8,17 @@ ext {
...
     // String property to store version name.
-    stringVersion = "6.10.2.0"
+    stringVersion = "6.10.3.0"
     // String property to store group id.
     stringGroupId = "com.google.ads.mediation"
 }
 
 android {
-    compileSdkVersion 30
+    compileSdkVersion 31
```
Comment (palrahul) [inline ThirdPartyAdapters/vungle/vungle/build.gradle:17]: as the adapter is still at API 30 and we cant bump that until Google wants to, does it even help bumping this on Test App side ?

**Review by palrahul — APPROVED** [review]
lgtm

**Review by palrahul — COMMENTED** [review]
@jiancchen what is the AdMob version with which this adapter version has been tested, can we please add that as well in the Change log.
Is that latest AdMob version supporting API 31, if not then do not think we can yet bump up API version on adapter

File: ThirdPartyAdapters/vungle/vungle/build.gradle [hunk 2]
```diff
@@ -31,10 +31,15 @@ android {
...
+    implementation ('com.github.vungle:vungle-android-sdk:6.10.3-RC3') {
         transitive=true
     }
 
+//    implementation ('com.vungle:publisher-sdk-android:6.10.3') {
+//        transitive=true
+//    }
+
     implementation 'androidx.annotation:annotation:1.2.0'
     implementation 'com.google.android.gms:play-services-ads:20.3.0'
```
Comment (palrahul) [inline ThirdPartyAdapters/vungle/vungle/build.gradle:44]: we should bump to use latest admob sdk dependency
https://developers.google.com/admob/android/sdk

**Comment by palrahul** [comment]
> @jiancchen what is the AdMob version with which this adapter version has been tested, can we please add that as well in the Change log. Is that latest AdMob version supporting API 31, if not then do not think we can yet bump up API version on adapter

NM, think AdMob seems to be fixing issues around Android 12 so possibly we can bump API version
https://developers.google.com/admob/android/rel-notes

File: ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java [hunk 3]
```diff
@@ -71,6 +71,9 @@ public void requestInterstitialAd(@NonNull Context context,
       return;
     }
 
+    VungleInitializer.getInstance()
+            .updateCoppaStatus(mediationAdRequest.taggedForChildDirectedTreatment());
+
```
Comment (anton-blashchuk-zoolatech) [inline ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java:76]: still relevant :
what about com.google.ads.mediation.vungle.rtb.VungleRtbInterstitialAd ? don't we need same lines there? and in com.google.ads.mediation.vungle.rtb.VungleRtbBannerAd too

**Review by anton-blashchuk-zoolatech — COMMENTED** [review]
other than question above looks good

File: ThirdPartyAdapters/vungle/vungle/build.gradle [hunk 4]
```diff
@@ -31,12 +31,17 @@ android {
 }
 
 dependencies {
-    implementation ('com.vungle:publisher-sdk-android:6.10.2') {
+
+    implementation ('com.github.vungle:vungle-android-sdk:6.10.3-RC3') {
```
Comment (HuFangshuai) [inline ThirdPartyAdapters/vungle/vungle/build.gradle:35]: If we want to release the GA build, please update this dependency. @jiancchen @palrahul

**Comment by HuFangshuai** [comment]
Please help to close this PR if not needed.

**Pull request closed without merging**